File: C++/Euler_3.cpp
#include <iostream>
using namespace std;
int main(){
   long num = 600851475143L, factor = 0L;
   for(int i = 1; i < 100000; i++){
      if(num / i == 0){
         for(int e = 1; e < 100000; e++){
            if((num/i)/e == 0){
            }
            else{   
               factor = num/i;
            }
         }
      }
   }
   std:: cout << factor << std::endl;
return 0;
}

File: Euler/Java/Euler_3.java
import java.util.*;

public class Euler_3{
   public static void main(String[] args){
      long factor = 0, factor2 = 0, num = 600851475143L;
      for(int i = 1; i < 200000000; i++){
         if(num % i == 0){
            for(int e = 1; e < 20000000; e++){
               if((num/i) % e == 0){
               }   
               else{
                  factor = num/i;
               }
            }
         }
      }
      System.out.print(factor + ", ");
   }
}